xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -111,7 +111,7 @@
      The &rpi; hardware does not have any built-in firmware. Its firmware is
      loaded from a special reserved FAT16 partition on the MicroSD card every
      time that the machine is powered on. There is also no battery-backed clock
-     or freely-modifiable "CMOS" memory to hold configuration information.
+     or non-volatile BIOS memory to hold configuration information.
```

Comment:
BIOS is also a PC-biased term - what about "non-volatile memory to hold bootloader configuration information"?

Reply:
It is, but this information is not solely restricted to or related to bootloaders.

The reason for this phrase is that it seems to be an accepted usage, e.g.  https://en.wikipedia.org/wiki/Nonvolatile_BIOS_memory

Reply:
Or, just call it non-volatile memory?

Reply:
Not ideal, but: done.